index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
> And then I wonder why rate limiting should be an optional part of the authenticator at all? Shouldn't we just require that authenticators limit brute force attacks?

Agreed. The recently-announced FIDO Authenticator certification program has a requirement for this -- see item 3.9 in [fido-authenticator-security-requirements
](https://fidoalliance.org/wp-content/uploads/fido-authenticator-security-requirements.pdf)

The previously-mentioned UAF authenticator commands "[security guidelines](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#security-guidelines)" was an apparent forebear to the [fido-authenticator-security-requirements
](https://fidoalliance.org/wp-content/uploads/fido-authenticator-security-requirements.pdf)

I'm hoping that we can in the W3C context point off to FIDO and/or other orgs as appropriate with regards to such authenticator details.